src/shared/calendar/calendar-navigator.tsx

```diff
@@ -0,0 +1,19 @@
+interface NavigatorProps {
+  next: string
+  prev: string
+  text: string
+  handlePrev: () => void
+  handleNext: () => void
+}
+
+/**이번달,이전달,이번년,다음년 이동하게 해주는 버튼 컴포넌트 */
+
+export default function CalendarNavigator({ next, prev, text, handlePrev, handleNext }: NavigatorProps) {
+  return (
+    <div className="flex">
+      <button onClick={handlePrev}>{prev}</button>
+      <h1>{text}</h1>
```

Comment:
P1

```suggestion
      <span>{text}</span>
```

h1 태그는 제목중 가장 중요한 제목을 말하기 때문에 위 방법은 시멘틱하다고 볼 수 없을것 같아요.